libGraphite/data/data.cpp

```diff
@@ -320,16 +307,12 @@ auto graphite::data::block::increase_size_to(std::size_t new_size) -> void
 
 auto graphite::data::block::clear() -> void
 {
-    set((uint32_t)0);
+    set((uint8_t)0, size());
 }
```

Comment:
This should not be needed, as it will ultimately pass through to `set(uint32_t)` anyway. If a difference in behaviour has been observed, then it should be the set function that is fixed, rather than this.

Reply:
This is related to the change to memset. Surface calls block::clear, which calls set(uint8_t), which calls memset. I observed a large reduction in time taken to initialise the surface for rlex.